src/spec/mod.rs

```diff
@@ -23,6 +23,10 @@
 
 use std::convert::From;
 
+mod fmt;
+#[allow(unused_imports)]
+pub use self::fmt::*;
```

Comment:
Also as a matter of style, we avoid glob imports.  Since all that file had is the one impl, I think that can just get inlined into this mod.

Reply:
I should have read this before my last comment, hah